skimage/morphology/selem.py

```diff
@@ -29,32 +32,46 @@ def square(width, dtype=np.uint8):
     return np.ones((width, width), dtype=dtype)
 
 
-def rectangle(width, height, dtype=np.uint8):
+@deprecate_kwarg({"height": "ncols", "width": "nrows"},
+                 removed_version="0.20.0")
+def rectangle(nrows=None, ncols=None, dtype=np.uint8):
```

Comment:
Specifying a default value is no more needed with the decorator
```suggestion
def rectangle(nrows, ncols, dtype=np.uint8):
```